lib/saputils.pm

```diff
@@ -81,4 +83,87 @@ sub calculate_hana_topology {
     return \%topology;
 }
 
+=head2 check_hana_topology
+    check_hana_topology([input => $saphanasr_showAttr_format_script_output]);
+
+    Expect the output of saputills::calculate_hana_topology as input.
+    Uses calculate_hana_topology to get a hash of hashes, and then
+    checks the output to make sure that the cluster is working and ready.
+
+    The checks performed are:
+    - All node_states are online
+    - All sync_states are either SOK or PRIM
+
+=cut
+
+
+sub check_hana_topology {
+    my (%args) = @_;
+    croak "Missing mandatory 'input' argument" unless $args{input};
+    record_info("cmd_out", $args{input});
```

Comment:
Won't this print a Perl structure, instead of readable information?
See: https://openqa.suse.de/tests/13779299#step/Stop_replica/71

Reply:
Yes indeed, it was a leftover from when I called the `calculate` function directly in this one. Replaced